sequencer/src/api/endpoints.rs

```diff
@@ -286,3 +289,35 @@ where
     )?;
     Ok(api)
 }
+
+pub(super) fn config<S, Ver: StaticVersionType + 'static>(_: Ver) -> Result<Api<S, Error, Ver>>
+where
+    S: 'static + Send + Sync + ReadState,
+    S::State: Send + Sync + HotShotConfigDataSource,
+{
+    let toml = toml::from_str::<toml::Value>(include_str!("../../api/config.toml"))?;
+    let mut api = Api::<S, Error, Ver>::new(toml)?;
+
+    let env_vars: Vec<(String, String)> = env::vars()
+        .filter(|(key, _)| key.starts_with("ESPRESSO_"))
```

Comment:
This feels a bit dangerous too because it's easy to include something private.

Reply:
Agree, I'm afraid here we also need a hard-coded list of the variables we want to pull :(

Reply:
I was thinking of adding a function to crate::Options and its modules for public env vars but then each bin also has options, and we might add some binaries in the future. How about a file containing all the variables for this endpoint?

Reply:
I like it